src/validation.cpp

```diff
@@ -3361,19 +3361,19 @@ bool CheckBlock(const CBlock& block, BlockValidationState& state, const Consensu
     return true;
 }
 
-void UpdateUncommittedBlockStructures(CBlock& block, const CBlockIndex* pindexPrev, const Consensus::Params& consensusParams)
+void ChainstateManager::UpdateUncommittedBlockStructures(CBlock& block, const CBlockIndex* pindexPrev) const
```

Comment:
nit: I'm not sure that this change is 100% necessary. The change seems to trade one reference for another, and since `ChainstateManager` is much larger in scope, it is harder to see after the change that this is a function that is limited in scope.

However, this is not at all a bit deal and up to subjective taste so feel free to ignore!

Reply:
It needs to call `DeploymentActive(.., SEGWIT)`, and that's something that the `ChainstateManager` is responsible for being able to answer. At least that was my logic.

Note that it's a `const` member function, so it's not doing anything to change the chain state, if that counts as reducing the scope?

Reply:
Sounds good!